src/vue/forms/LoginForm.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <form class="app-form auth-form" @submit.prevent="submit">
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          v-model="form.email"
+          @blur="touchField(`form.email`)"
+          id="login-email"
+          :label="globalize('auth-pages.email')"
+          :error-message="errorMessage(`form.email`)"
+        />
+      </div>
+    </div>
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          v-model="form.password"
+          @blur="touchField(`form.password`)"
```

Comment:
I’d recommend using plain `$v.form.password.$touch` here instead because of redundant incapsulating

Reply:
resolved
it is not possible to test then, $touch is read-only so we cannot stub/spy on it